app/views/restaurants/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Restaurants</h1>
+
+<% @restaurants.each do |restaurant| %>
+
+  <p>
+    <span><%= link_to restaurant.name, restaurant_path(restaurant) %></span>
+    <span><% restaurant.street %></span>
```

Comment:
🔨 @NancyGoodman the directions called for only the restaurant name (and the link) to be listed in the index view, the restaurant's address should only be viewable on the show page of each restaurant.